taxcalc/functions.py

```diff
@@ -1641,8 +1637,8 @@ def IITAX(c09200, c59660, c11070, c10960, c11600, c10950, _eitc, e11580,
 
     _refund = _refund + partially_refundable_CTC
 
-    _payments = (c59660 + c10950 + c10960 + c11070 + e10000 + e11550 + e11580 +
-                 e11450)
+    _payments = (c59660 + c10950 + c10960 + c11070 + e10000 + e11550 +
+                 e11450 + e11500)
```

Comment:
`e11500` is zero. `e11580`, First Time Homebuyer Credit, line 60b in 1040, is part of other tax, instead of payments.